### PR TITLE
Added flag for generation of python init files.

### PR DESCRIPTION
**Pull request by briantenazas**

### CODE REVIEW

File: all/entrypoint.sh [hunk 1]
```diff
@@ -15,9 +15,11 @@ printUsage() {
     echo " -o DIRECTORY         The output directory for generated files. Will be automatically created."
     echo " -i includes          Extra includes"
     echo " --with-gateway       Generate grpc-gateway files (experimental)."
+    echo " --no-init            Does not generate __init__.py files (Python 3.6 and up)"
```
Comment (mdkess) [inline all/entrypoint.sh]: Add a test to all/test.sh

File: all/entrypoint.sh [hunk 2]
```diff
@@ -130,14 +141,14 @@ fi
 # If __init__.py files are needed at higher level directories (i.e.
 # directories above $OUT_DIR), it's the caller's responsibility to
 # create them.
-if [[ $GEN_LANG == "python" ]]; then
+if [[ $GEN_LANG == "python" && $ $NO_INIT = false ]]; then
```
Comment (mdkess) [inline all/entrypoint.sh]: This extra condition seems like a typo?

**Comment by mhamrah** [comment]
is this still relevant?

**Comment by designjockey** [comment]
Stale PR, closing. Feel free to re-open if needed.

**Pull request closed without merging**